modules/javafx.graphics/src/main/java/javafx/css/Selector.java

```diff
@@ -42,6 +42,13 @@
  */
 abstract public class Selector {
 
+    /**
+     * @deprecated This constructor was exposed erroneously and will be removed in the next version.
+     */
```

Comment:
Please direct the reader to the way of obtaining an instance of this class like you did for `ShapeConverter`.

Reply:
Probably you can refer to the `createSelector` method.